src/components/Categories.js

```diff
@@ -0,0 +1,7 @@
+const Category = () => (
+  <div>
+    <h1>Categories</h1>
+  </div>
```

Comment:
- [ ] The `Category` component should only have a button with the text "check status" as per project requirements

Reply:
done

Reply:
This change has been done well